markdown/pxf/ReadWritePXF.html.md.erb

```diff
@@ -131,6 +149,8 @@ Note: The <code class="ph codeph">DELIMITER</code> parameter is mandatory.
 </tbody>
 </table>
 
+**Notes**: Metadata identifies the Java class that provides field definitions in the relation. OutputFormat identifies the file format for which a specific profile is optimized. While the built-in `Hive*` profiles provide Metadata and OutputFormat classes, most profiles will have no need to implement or specify these classes.
```

Comment:
the statement is kind of vague, yes.  i will make it clearer.  

it seems to me that the outputFormat property is more admin- or developer-focused.  what i mean here is that a typical end-user using a Hive* profile probably does not need to know about the setting.  an admin configuring custom profiles or a developer creating a custom plug-in may be interested.  the current admin- and developer- focused PXF docs need some attention.  can we address a more detailed discussion of this property as part of that rework at a later time?